pkg/apiserver/resthandler.go

```diff
@@ -684,15 +686,29 @@ func DeleteResource(r rest.GracefulDeleter, checkBody bool, scope RequestScope,
 // to use it.
 // TODO: add appropriate structured error responses
 func queryToObject(query url.Values, scope RequestScope, kind string) (runtime.Object, error) {
```

Comment:
@deads2k #17163 it's is using the ServerAPIVersion https://github.com/kubernetes/kubernetes/pull/17163/files#diff-fe6dbe5fa548ea5c031a5be1669a6376R242